File: app/src/main/java/com/wang17/myclock/database/utils/DayItem.java
package com.wang17.myclock.database.utils;

import java.util.UUID;

/**
 * Created by Administrator on 2017/10/11.
 */

public class DayItem {
    private UUID id;
    private String name;
    private String summary;
    private int targetInHour;

    public DayItem() {
        this.id = UUID.randomUUID();
    }

    public DayItem(UUID id) {
        this.id = id;
    }

    public DayItem(String name, String summary, int targetInHour) {
        this();
        this.name = name;
        this.summary = summary;
        this.targetInHour = targetInHour;
    }

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSummary() {
        return summary;
    }

    public void setSummary(String summary) {
        this.summary = summary;
    }

    public int getTargetInHour() {
        return targetInHour;
    }

    public void setTargetInHour(int targetInHour) {
        this.targetInHour = targetInHour;
    }
}

File: app/src/main/java/com/wang17/myclock/database/utils/DayItem.kt
package com.wang17.myclock.database.utils

import java.util.*

/**
 * Created by Administrator on 2017/10/11.
 */
class DayItem {
    var id: UUID= UUID.randomUUID()
    var name: String = ""
    var summary: String = ""
    var targetInHour = 0

    constructor(id: UUID) {
        this.id = id
    }

    constructor(name: String, summary: String, targetInHour: Int) {
        this.name = name
        this.summary = summary
        this.targetInHour = targetInHour
    }
}